.github/workflows/static-analysis.datadog.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches: [master]
```

Comment:
@skarimo was curious why you're choosing to only run on the master branch? If you run on any branch you'd be able to get PR comments on your PRs for any branch

Reply:
As discussed offline majority of this repo is generated and we do not get huge benefit from running it per pr basis